solutions/swb-reference/config/jest.config.json

```diff
@@ -19,7 +19,9 @@
     "<rootDir>/src/postDeployment.ts",
     "<rootDir>/src/services/dataSetService.ts",
     "<rootDir>/src/services/projectEnvTypeConfigService.ts",
-    "<rootDir>/src/services/keyPairService.ts"
+    "<rootDir>/src/errors/databaseError.ts",
```

Comment:
do you still need these if they only live in `swb-app`?

Reply:
Nice catch. Yup!